docs/content/latest/contribute/core-database/run-unit-tests.md

```diff
@@ -86,3 +87,30 @@ org.yb.client.TestYBClient.txt
 {{< note title="Note" >}}
 The YB logs are contained in the output file now.
 {{< /note >}}
+
+### YSQL Java tests
+
+YSQL java tests are in `java/yb-pgsql/src/test/java/org/yb/pgsql/`.  They can be run as:
+```bash
+./yb_build.sh --java-test TestPgTruncate
+```
+Some of those tests, `TestPgRegress*` use the postgres regress test framework: `src/postgres/src/test/regress`.  
+They should each correspond to a schedule (e.g. `java/yb-pgsql/src/test/java/org/yb/pgsql/TestPgRegressArrays.java` references `src/postgres/src/test/regress/yb_arrays_schedule`) 
+that is run by our modified version of `pg_regress`.
+
+Each schedule has a serial order of files to run.  For example, the `yb_arrays_schedule` will first run `build/latest/postgres_build/src/test/regress/sql/yb_pg_int8.sql` 
```

Comment:
Because this has a chance of becoming outdated, explain why: the first `test:` line in `yb_arrays_schedule` is `test: yb_pg_int8`.

Reply:
I've commited all changes. Please quick review and I commit.